Tests/SentryTests/Protocol/SentryLogAttributeTests.swift

```diff
@@ -0,0 +1,201 @@
+        let attribute = SentryLog.Attribute.boolean(true)
+        
+        let data = try JSONEncoder().encode(attribute)
+        let json = try JSONSerialization.jsonObject(with: data) as? [String: Any]
+        
+        XCTAssertNotNil(json)
+        XCTAssertEqual(json?["type"] as? String, "boolean")
+        XCTAssertEqual(json?["value"] as? Bool, true)
+    }
+    
+    func testEncodeIntegerAttribute() throws {
+        let attribute = SentryLog.Attribute.integer(42)
+        
+        let data = try JSONEncoder().encode(attribute)
+        let json = try JSONSerialization.jsonObject(with: data) as? [String: Any]
+        
+        XCTAssertNotNil(json)
+        XCTAssertEqual(json?["type"] as? String, "integer")
+        XCTAssertEqual(json?["value"] as? Int, 42)
+    }
+    
+    func testEncodeDoubleAttribute() throws {
+        let attribute = SentryLog.Attribute.double(3.14159)
+        
+        let data = try JSONEncoder().encode(attribute)
+        let json = try JSONSerialization.jsonObject(with: data) as? [String: Any]
+        
+        XCTAssertNotNil(json)
+        XCTAssertEqual(json?["type"] as? String, "double")
+        XCTAssertEqual(json?["value"] as! Double, 3.14159, accuracy: 0.00001)
+    }
+    
+    // MARK: - Decoding Tests
+    
+    func testDecodeStringAttribute() throws {
+        let json = Data("""
+        {
+            "type": "string",
+            "value": "decoded value"
+        }
+        """.utf8)
+        
+        let attribute = try XCTUnwrap(decodeFromJSONData(jsonData: json) as SentryLog.Attribute?)
+        
+        XCTAssertEqual(attribute.type, "string")
+        XCTAssertEqual(attribute.value as? String, "decoded value")
+    }
+    
+    func testDecodeBooleanAttribute() throws {
+        let json = Data("""
+        {
+            "type": "boolean",
+            "value": false
+        }
+        """.utf8)
+        
+        let attribute = try XCTUnwrap(decodeFromJSONData(jsonData: json) as SentryLog.Attribute?)
+        
+        XCTAssertEqual(attribute.type, "boolean")
+        XCTAssertEqual(attribute.value as? Bool, false)
+    }
+    
+    func testDecodeIntegerAttribute() throws {
+        let json = Data("""
+        {
+            "type": "integer",
+            "value": 12345
+        }
+        """.utf8)
+        
+        let attribute = try XCTUnwrap(decodeFromJSONData(jsonData: json) as SentryLog.Attribute?)
+        
+        XCTAssertEqual(attribute.type, "integer")
+        XCTAssertEqual(attribute.value as? Int, 12_345)
+    }
+    
+    func testDecodeDoubleAttribute() throws {
+        let json = Data("""
+        {
+            "type": "double",
+            "value": 2.71828
+        }
+        """.utf8)
+        
+        let attribute = try XCTUnwrap(decodeFromJSONData(jsonData: json) as SentryLog.Attribute?)
+        
+        XCTAssertEqual(attribute.type, "double")
+        XCTAssertEqual(attribute.value as! Double, 2.71828, accuracy: 0.00001)
```

Comment:
`m`: Please don't use `as!` in tests. When this fails the failure message can be cryptic and it aborts the tests. Please use `XCTUnwrap` instead.
```suggestion
        let doubleValue = try XCTUnwrap(attribute.value as? Double)
        XCTAssertEqual(doubleValue, 2.71828, accuracy: 0.00001)
```